appcues/src/main/java/com/appcues/action/ActionKoin.kt

```diff
@@ -14,7 +14,7 @@ internal object ActionKoin : KoinScopePlugin {
 
     override fun ScopeDSL.install() {
         scoped { ActionRegistry(scope = get(), logcues = get()) }
-        scoped { ActionProcessor(appcues = get(), appcuesCoroutineScope = get()) }
+        scoped { ActionProcessor(scope = get(), appcuesCoroutineScope = get()) }
```

Comment:
this one (and the line above) - if we we are passing the scope in to avoid circular dependencies - maybe we should just not pass any other koin scope dependencies in any longer? Then resolve them as injected props with scope.inject()? It feels strange to have a mix in the same class.

Reply:
will do thanks